src/kilo.c

```diff
@@ -939,13 +939,45 @@ void editorMoveCursorToRowEnd() {
     }
     int rowlen = row ? row->size : 0;
     int size = rowlen - E.cx-E.coloff;
-    
-    while (size != 0 ) {
+
+    while (size != 0) {
         editorMoveCursor(RIGHT);
         size = size -1;
     }
 }
 
+int editorMoveCursorToFirst(char c) {
```

Comment:
I've been following the UNIX convention of non-zero return meaning failure. Just for consistency, would you mind following suit? Also, once you rebase you can just use bool, true & false instead of int, 1 & 0.